Dynatrace/dynatrace-operator
# Add GitHub action to automatically update renovate config

## Description

Currently we have to manually edit the renovate config file to always point to the last 3 releases.
This PR automates the process and creates a github action that creates a PR that updates the file.

The implementation uses node as the official reference library for JSON5 parsing was implemented in node. 

## How can this be tested?

Create a release branch and check if the PR for `.github/renovate.json5` was created.

## Checklist
